plugin/pkg/admission/imagepolicy/admission.go

```diff
@@ -181,6 +181,20 @@ func (a *Plugin) admitPod(pod *api.Pod, attributes admission.Attributes, review
 		return errors.New("one or more images rejected by webhook backend")
 	}
 
+	if review.Status.Annotations == nil {
+		return nil
+	}
+	reviewAnnotations := a.filterAnnotations(review.Status.Annotations)
+	if len(reviewAnnotations) > 0 {
+		annotations := pod.GetAnnotations()
+		if annotations == nil {
+			annotations = make(map[string]string)
+		}
+		for k, v := range reviewAnnotations {
+			annotations[k] = v
+		}
+		pod.ObjectMeta.SetAnnotations(annotations)
```

Comment:
This function is called from both Admit & Validate, but Validate is non-mutating. This should be moved to an Admit-only piece, unless you decide to move the annotation to the audit log.